jwst/pipeline/tests/test_calwebb_spec2.py

```diff
@@ -0,0 +1,63 @@
+"""Test calwebb_spec2"""
+
+from contextlib import contextmanager
+from os import path
+import pytest
+
+from ...associations.asn_from_list import asn_from_list
+from ...associations.lib.rules_level2_base import DMSLevel2bBase
+from ...datamodels import open as dm_open
+from ..calwebb_spec2 import Spec2Pipeline
+
+
+def abspath(filepath):
+    """Get the absolute file path"""
+    return path.abspath(path.expanduser(path.expandvars(filepath)))
+
+
+DATAPATH = abspath(
+    '$DEVDIR/testdata/jwst_data/dev/build7.1/spec2_test'
+)
+EXPFILE = 'jw00035001001_01101_00001_mirimage_rate.fits'
```

Comment:
No, not all. MIRI MRS spectra use their own dedicated detectors (IFUSHORT, IFULONG). NIRISS and NIRCam, on the other hand, project both direct imaging and grism spectra onto the same detectors (like MIRI imaging and LRS spectra).